SU2_CFD/include/variables/CNEMOEulerVariable.hpp

```diff
@@ -81,6 +81,7 @@ class CNEMOEulerVariable : public CVariable {
   LAM_VISC_INDEX, EDDY_VISC_INDEX, nSpecies;
 
   su2double Tve_Freestream; /*!< \brief Freestream vib-el temperature. */
+  bool implicit;
```

Comment:
```suggestion
  const bool implicit;
```
You can make it const and set it in the initializer list of the class